linera-execution/src/runtime.rs

```diff
@@ -401,10 +401,16 @@ impl SyncRuntimeInternal<UserContractInstance> {
             }
             #[cfg(not(web))]
             hash_map::Entry::Vacant(entry) => {
-                let (code, description) = self
+                let txn_tracker_moved = mem::take(&mut self.transaction_tracker);
```

Comment:
(We really need to find a better way to handle the transaction tracker. Not in this PR, of course!)